src/target/riscv/riscv-013.c

```diff
@@ -1650,11 +1651,13 @@ static int init_target(struct command_context *cmd_ctx,
 	generic_info->set_register = &riscv013_set_register;
 	generic_info->select_current_hart = &riscv013_select_current_hart;
 	generic_info->is_halted = &riscv013_is_halted;
-	generic_info->halt_current_hart = &riscv013_halt_current_hart;
+	generic_info->halt_go = &riscv013_halt_go;
```

Comment:
Line 1654 appears to be duplicated on line 1660, I'm assuming only one of those lines is necessary?

Reply:
Good catch. Fixed.